packages/url-launcher/README.md

```diff
@@ -41,10 +41,12 @@ URL schemes depend on the underlying platform and installed apps.
 
 Common schemes supported by both iOS and Android:
 
-* http:<URL> , https:<URL>
-* mailto:<email-address>
-* tel:<phone-number>
-* sms:<phone-number>
+| Scheme | Action | Example |
+|---|---|---|
+| `http:<URL>` , `https:<URL>` | `http://flutter.io` | Open URL in the default browser |
+| `mailto:<email address>` | `'mailto:smith@example.org?subject=Help&body=I need help!'` | Open <email address> in the default email app |
```

Comment:
Change the mailto one to just `'mailto:smith@example.org'` until we figure out what is going on with iOS